contrib/openstack/config.sh

```diff
@@ -0,0 +1,55 @@
+export DEIS_TEST_DOMAIN="xip.io"
+export GLANCE_IMAGE
+export FLAVOR
+export KEY
+export NETWORK
+export POOL
+export OS_USERNAME
+export OS_PASSWORD
+export OS_TENANT_NAME
+export OS_AUTH_URL
+
+prompt "What glance image should I use? [coreos-stable]" GLANCE_IMAGE
+prompt "What flavor ? [2]" FLAVOR
+prompt "Path to your public key: [~/.ssh/id_rsa.pub] " KEY
+prompt "What is the name of your neutron network? [internal]" NETWORK
+prompt "What is the name of your floating IP pool ? [external]" POOL
+prompt "what is your openstack auth URL? [${OS_AUTH_URL}]" OS_AUTH_URL
+prompt "what is your openstack username? [${OS_USERNAME}]" OS_USERNAME
+prompt "what is your openstack password? [**********]" OS_PASSWORD
+prompt "what is your openstack tenant name? [${OS_TENANT_NAME}]" OS_TENANT_NAME
```

Comment:
Prompt takes 3 variables:

```
prompt "<question>" <return_var> <default suggested var>
```

or just look at the [implementation here](https://github.com/deis/rigger/blob/master/lib/interactive.sh#L52).